Add a way to get project version

techncl opened the pull request.

MancunianSam commented on src/main/scala/uk/gov/nationalarchives/Main.scala:
```diff
@@ -49,6 +52,16 @@ object Main extends IOApp {
+            val versionWithoutSnapshot: String = BuildInfo.version.stripSuffix("-SNAPSHOT")
+            versionWithoutSnapshot.getBytes("UTF-8")
+          }
+          .through(Files[IO].writeAll(Path(s"${config.versionPath}-${UUID.randomUUID()}")))
```
You need a forward slash here. The config path will be something like `/tmp/versions` and the file will be `/tmp/versions/eb75effa-bd97-4312-949d-9eec179fa1e3` so this needs to be `s"${config.versionPath}/${UUID.randomUUID()}"`

MancunianSam reviewed (commented):
Just one minor comment thanks. I think you need to put a version in version.sbt to start with so if you just put `ThisBuild / version := "0.0.1-SNAPSHOT"` in there.

techncl commented:
> Just one minor comment thanks. I think you need to put a version in version.sbt to start with so if you just put `ThisBuild / version := "0.0.1-SNAPSHOT"` in there.

Ah, I was so confused when I read your comment because I remember that was the first thing I added but then I realised that I did a merge but decided to abort it and IntelliJ went weird and deleted a bunch of stuff I added prior 🤦
